actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -26,7 +26,42 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      class TagBuilder #:nodoc:
+        include TagHelper
```

Comment:
Roger that. There is one more detail to figure out:
Should we keep them (`content_tag_string`, `tag_options`) private?
Is there a way to put them into the builder class, make `tag` and `content_tag` use them, but still keep them private?